app/scripts/views/mixins/migration-mixin.js

```diff
@@ -8,10 +8,24 @@
 define(function (require, exports, module) {
   'use strict';
 
-
   module.exports = {
-    isMigration: function () {
-      return this.relier.has('migration');
+    isSyncMigration: function isSyncMigration () {
+      var isMigration = this.relier.has('migration');
```

Comment:
I feel like there's a lot of code here to handle normalization. Can the normalizing be pushed to the relier, at least lower case the `migration` value so consumers can ignore that aspect?

If so, we could collapse this to:

```js
return this.relier.get('migration') === `sync11`;
```

As an aside, we should move the `amo` and `sync11` constants out of this file into [lib/constants.js](https://github.com/mozilla/fxa-content-server/blob/0c84ed2695abf0411c875aa499f61be38e191464/app/scripts/lib/constants.js#L42)